index.html

```diff
@@ -266,137 +289,324 @@ <h4 class="packcage-title">Package Summery Details</h4>
+      <div class="container">
+        <h2 class="title-h2">Other Packages</h2>
+        <div class="row">
+          <div class="col-md-8">
+            <div class="hotel-card">
+              <picture class="hotel-card__img">
+                <source srcset="./images/pk_1.webp 1x, 
+                ./images/pk_1_2x.webp 2x" type="image/webp" class="hotel-card__img">
+                <source srcset="./images/pk_1.jpg 1x, 
+                ./images/pk_1_2x.jpg 2x" type="image/jpeg" class="hotel-card__img">
+                <img src="./images/pk_1.jpg" alt="hotel blue heaven" class="hotel-card__img">
+              </picture>
+              <span class="hotel-card__flash">Flash Offer</span>
+              <div class="hotel-card__info">
+                <div class="hotel-stars">
+                  <span class="icon icon-gold-star gold-star"></span>
+                  <span class="icon icon-gold-star gold-star"></span>
+                  <span class="icon icon-gold-star gold-star"></span>
+                  <span class="icon icon-gold-star gold-star"></span>
+                  <span class="icon icon-gold-star gold-star"></span>
+                </div>
+                <h4 class="hotel-card__title">Hotel Blue Haven</h4>
+                <p class="hotel-card__text">
+                  Aute quis duis excepteur excepteur ipsum cat eiusmod consectetur enim laborum magna llit.
+                  Ipsum est fugiat velit ea llamco do esse ut in veniam sun in onsequat. Aute quis duis epteur
+                  excepteur ipsum occaecat eiusmod nsectetur enim laborum magna mollit. Ipsum est fugiat velit
+                  ea ullamco do
+                </p>
+                <div class="hotel-card__service">
+                  <span class="icon icon-map-pin hotel-card__icon">
+                    <span class="icon-text">1749 Wheeler Ridge Delaware</span>
+                  </span>
+                  <span class="icon icon-user hotel-card__icon">
+                    <span class="icon-text">2 x Guests</span>
+                  </span>
+                  <span class="icon icon-house hotel-card__icon">
+                    <span class="icon-text">1 x Room</span>
+                  </span>
+                </div>
+                <span class="hotel-card__old-price">10,500$</span>
+                <div class="price-btn">
+                  <span class="price">8,500$</span>
+                  <button class="book-btn">Book Now</button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <div class="hotel-card hotel-card--sm">
+              <picture class="hotel-card__img hotel-card__img--sm">
+                <source srcset="./images/hotel_belle_1x.webp 1x, 
+                                ./images/hotel_belle_2x.webp 2x" type="image/webp"
+                  class="hotel-card__img hotel-card__img--sm">
+                <source srcset="./images/hotel_belle_1x.jpg 1x, 
```

Comment:
The same. Please, check all the document.

Reply:
> The same. Please, check all the document.

fixed